keps/sig-scheduling/5075-dra-consumable-capacity/README.md

```diff
@@ -0,0 +1,1150 @@
+  the total allocated capacity is calculated by aggregating the ConsumedCapacities from all resource claims's `DeviceRequestAllocationResult` that have already been allocated.
+- Before allocation proceeds, existing selection criteria (defined by `alloc.isSelectable`) are evaluated. 
+  These include the class selector and request selector.
+- A new `device.shareable` key is introduced in the CEL selector,
+  enabling policies and constraints to recognize whether a device supports shareable allocation.
+- If a device is considered selectable, the `CmpRequestOverCapacity` function is invoked to verify 
+  whether the consumed capacity would exceed the device's remaining capacity. 
+  The remaining capacity is calculated based on the sum of already allocated and currently allocating capacities.
+  - consumed capacity is derived from the requested amount specified in the resource claim, adjusted by the device’s capacity sharing policy, if defined.
+  - This value may differ from the originally requested amount—it is rounded up to the nearest valid capacity according to the policy (e.g., using Minimum + ⌈(Requested - Minimum)/ChunkSize⌉ × ChunkSize logic).
+- If the device has enough remaining capacity to satisfy the consumed amount, constraint checks are applied. 
+  In addition to the existing MatchAttribute, this proposal introduces a new constraint: `DistinctAttribute`, which ensures attribute uniqueness across allocated devices.
+- Once all selection and constraint checks pass, the allocation is valid. The allocation result is updated with:
+  - The shareable allocation identifier, which uniquely identifies the allocation on a shareable device.
+  - The calculated consumed capacity, if a capacity sharing policy was applied.
+    This consumed capacity is tracked as part of the device’s `allocatingCapacity`, 
+    allowing it to be included in remaining capacity calculations for future allocations within the same call.
+- Finally, the shareable allocation identifiers and consumed capacities from all internal results
+  are propagated to the DeviceRequestAllocationResult.
+
+### Examples
+
+#### Shareable DeviceClass's selector
+
+```yaml
+selectors:
+  - cel:
+      expression: |-
+        device.shareable == true
+```
+
+#### ResourceClaim with capacity requirement
+
+```yaml
+kind: ResourceSlice
+...
+spec:
+  driver: guaranteed-cni.dra.networking.x-k8s.io
+  devices:
+  - name: eth1
+    basic:
+      shareable: true
+      attributes:
+        name:
+          string: "eth1"
+      capacity:
+        bandwidth:
+          sharingPolicy:
+            range:
+              minimum: "1Mi"
+              chunkSize: "8"
+          value: "10Gi"
+```
+
+#### ResourceClaim's request
+
+```yaml
+kind: ResourceClaim
+...
+spec:
+  devices:
+    requests:
+    - name: nic
+      deviceClassName: qos-aware-shared.device.x-k8s.io
+      capacities:
+        minimum:
+          bandwidth: 5Gi
```

Comment:
How is this name handled? In the Go above, it's a `QualifiedName`. So, if it's not qualified here, what domain do we use to qualify it? Does it vary based on the underlying device being looked at (uses the domain from that device)? Or is it `k8s.io` or equivalent?